spec/controllers/static_pages_controller_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe StaticPagesController, type: :controller do
```

Comment:
Consider moving to request specs instead of controller specs. Controller specs have been deprecated and request specs load the application in a proper way to check to make sure the controller actions are working correctly.

https://github.com/rspec/rspec-rails/issues/1838